actors/builtin/reward/reward_actor.go

```diff
@@ -94,12 +94,12 @@ func (a Actor) AwardBlockReward(rt vmr.Runtime, params *AwardBlockRewardParams)
 	AssertMsg(big.Add(rewardPayable, penalty).LessThanEqual(priorBalance),
 		"reward payable %v + penalty %v exceeds balance %v", rewardPayable, penalty, priorBalance)
 
-	_, code := rt.Send(minerAddr, builtin.MethodsMiner.AddLockedFund, &rewardPayable, rewardPayable)
-	builtin.RequireSuccess(rt, code, "failed to send reward to miner: %s", minerAddr)
+	// if this fails, we can assume the miner is responsible and do not to fail here
+	_, _ = rt.Send(minerAddr, builtin.MethodsMiner.AddLockedFund, &rewardPayable, rewardPayable)
```

Comment:
Let's log this failure with `rt.Log`.

Reply:
Also, @anorth, should we burn the funds if we fail to send them? What happens otherwise?